db/models.py

```diff
@@ -1,35 +1,48 @@
+from typing import Any
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+from django.core.exceptions import ValidationError
 
 
 class Genre(models.Model):
-    name = models.CharField(max_length=255, unique=True)
+    name: str = models.CharField(max_length=255, unique=True)
```

Comment:
Type annotations for Django model fields are not necessary and could lead to confusion. It's better to use Django's field types directly without type annotations.